servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -179,6 +212,18 @@ public int deleteExpiredX509CertRecords(int expiryTimeMins) {
         return 0;
     }
 
+    @Override
```

Comment:
Our new notification is only for ZTS so at this point these methods are not required.